vite.extension.config.mts

```diff
@@ -66,7 +66,7 @@ export default defineConfig((env) => {
                 js: ['extension/src/frontend/index.ts'],
               },
             ],
-            permissions: ['webNavigation', 'storage'],
+            permissions: ['webNavigation', 'storage', 'tabs'],
```

Comment:
The `tabs` permission is needed to read the `pendingUrl` and `url` properties on tabs.